### PR TITLE
Added VEGA in asset pool to TVL, added methodology, added Vesting

### PR DESCRIPTION
**Pull request by C0deMunk33**
Added VEGA in asset pool to TVL, added methodology, added Vesting.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/vega-protocol exports TVL: 
        
 
 
    ethereum-vesting          43.74 M
    ethereum-staking          43.43 M
    ethereum                  3.61 M
    vesting                   43.74 M
    staking                   43.43 M
    
    total                    3.61 M

File: projects/vega-protocol/index.js [hunk 1]
```diff
@@ -36,10 +37,14 @@ Object.keys(config).forEach(chain => {
         fromBlock,
       })
       const blacklistedTokens = []
-      if (vega) blacklistedTokens.push(vega)
```
Comment (g1nt0ki) [inline projects/vega-protocol/index.js:39]: sorry, this needs to be retained, we dont count count own tokens as tvl as it costs the protocol nothing to inflate tvl.

File: projects/vega-protocol/index.js [hunk 2]
```diff
@@ -57,8 +62,19 @@ Object.keys(config).forEach(chain => {
         '0xcb84d72e61e383767c4dfeb2d8ff7f4fb89abc6e': BigNumber(vegaStakedInVesting).plus(BigNumber(vegaStakedInStaking)).toFixed(0)
```
Comment (g1nt0ki) [inline projects/vega-protocol/index.js:62]: okay, I am seeing the same tokens counted in two places (staking & vesting) (?)

The mistake might be on my part for including it in staking, are these tokens in circulation, or they being vested and investors/team get their hands on it for the first time after they get unlocked at certain time?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by g1nt0ki** [comment]
closing because of inactivity

**Pull request closed without merging**